tests/test_lint_makefile.py

```
@@ -24,39 +29,75 @@ def test_spaces_after_multiline_continuation(capsys):
     )
-    ast = parser.parse(io.StringIO(mk))
-    targets, deps, dep_map = lint_makefile.parse_targets(ast)
-    valid = lint_makefile.validate(mk.split('\n'), targets, deps, dep_map)
-    captured = capsys.readouterr()
+    valid, errors = run_validation(mk)
     assert not valid
-    assert "Space instead of tab" in captured.out
+    assert any(err.error_type == "space instead of tab" for err in errors)
+
+
+def test_error_includes_line_info():
```

Ruff (0.11.9)

Missing return type annotation for public function `test_error_includes_line_info`

Add return type annotation: `None`

(ANN201)